services/QuillLMS/engines/evidence/spec/models/evidence/activity_spec.rb

```diff
@@ -32,6 +32,21 @@ module Evidence
 
       it { should validate_length_of(:scored_level).is_at_most(100) }
 
+      context 'version' do
+        it 'a monotonically increasing version should be valid' do
+          activity = create(:evidence_activity)
+          expect { activity.update!({title: activity.title << " 2"}) }
+          .to change { activity.version }.by 1
+        end
+
+        it 'a non-monotonically increasing version should be invalid' do
+          activity = create(:evidence_activity)
+          expect do
+            activity.update!({ version: activity.version + 2 })
+          end.to raise_error(ActiveRecord::RecordInvalid)
+        end
+      end
+
```

Comment:
These tests look great.  Is it worth having some kind of test around the `before_validation` hook?

Reply:
n/a - I removed this hook